src/supaplex.c

```diff
@@ -961,6 +962,9 @@ void buildScalingModeOptionTitle(char output[kMaxAdvancedOptionsMenuEntryTitleLe
         case ScalingModeFullscreen:
             mode = kFullscreenScalingModeString;
             break;
+        case ScalingModeAspectCorrect:
+            mode = kAspectCorrectScalingModeString;

```

Comment:
I think calling it "correct" is kind of misleading or at least confusing. I'd rather have something more literal like:
```suggestion
        case ScalingModeAspect43:
            mode = kAspect43ScalingModeString;
```
Or maybe:
```suggestion
        case ScalingModeAspect4_3:
            mode = kAspect4_3ScalingModeString;
```

It doesn't look as good, but I think I prefer it 😂

Reply:
For reference, those names (aspect fit and aspect fill) come from Apple's UIKit terminology (I've been an iOS dev for ~10 years):
- https://developer.apple.com/documentation/uikit/uiviewcontentmode/uiviewcontentmodescaleaspectfit
- https://developer.apple.com/documentation/uikit/uiviewcontentmode/uiviewcontentmodescaleaspectfill

Both of them take as much screen as possible and keep the aspect ratio, with these differences:
- Aspect fill doesn't leave any black area on the screen (so part of the content is cropped if needed).
- Aspect fit leaves black areas so the content is not cropped.

![image](https://user-images.githubusercontent.com/1083228/111765985-4abfd780-88a5-11eb-9de1-7833d3858c8f.png)

What you're implementing here would be like aspect fit, but just a different aspect ratio. I understand where "correct" comes in this case, but still sounds kind of biased towards what people think was correct 🤔 I'd rather stick to something more specific about those 4:3, but maybe do this:

`ScalingModeAspectCorrect` -> `ScalingModeAspectFit4_3`
`kAspectCorrectScalingModeString` -> `kAspectFit4_3ScalingModeString`

Or even better, if that `4_3` bothers you as much as it bothers me 😂 We can go with CRT:

`ScalingModeAspectCorrect` -> `ScalingModeAspectFitCRT`
`kAspectCorrectScalingModeString` -> `kAspectFitCRTScalingModeString`